.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,297 @@
+            $env:Path += ";C:\Program Files (x86)\Microsoft Visual Studio\2019\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+          } else {
+            Write-Host "Unknown image name: '$($env:TILEDB_GA_IMAGE_NAME)'"
+            exit $LastExitCode
+          }
+          
+          # allow double-checking path
+          cmd /c "echo $PATH"
+
+          # TODO DEBUG move this back in the TILEDB_S3 section
+          # currently we do not run S3 tests on Windows because tests time out (minio+azure slow?)
+          #& "$env:BUILD_SOURCESDIRECTORY\scripts\install-minio.ps1"
+
+          $bootstrapOptions = "-EnableVerbose -EnableStaticTileDB -EnableBuildDeps"
+          if ($env:TILEDB_S3 -eq "ON") {
+            $bootstrapOptions = "-EnableS3 " + $bootstrapOptions
+          }
+          if ($env:TILEDB_AZURE -eq "ON") {
+            $bootstrapOptions = "-EnableAzure " + $bootstrapOptions
+          }
+          if ($env:TILEDB_GCS -eq "ON") {
+            $bootstrapOptions = "-EnableGCS " + $bootstrapOptions
+            #NOTE: GCS simulator not yet actually in place.
+          }
+          if ($env:TILEDB_SERIALIZATION -eq "ON") {
+            $bootstrapOptions = "-EnableSerialization " + $bootstrapOptions
+          }
+          # static already added above as initial default
+          # if ($env:TILEDB_STATIC -eq "ON") {
+          #  $bootstrapOptions = $bootstrapOptions + " -EnableStaticTileDB"
+          # }
+          # if ($env:TILEDB_HDFS -eq "ON") {
+          #  $bootstrapOptions = $bootstrapOptions + " -EnableHDFS"
+          #}
+          if ($env:TILEDB_TOOLS -eq "ON") {
+            $bootstrapOptions = $bootstrapOptions + " -EnableTools"
+          }
+          if ($env:TILEDB_DEBUG -eq "ON") {
+            $bootstrapOptions = $bootstrapOptions + " -EnableDebug"
+          }
+          # if ($env:TILEDB_CI_ASAN -eq "ON") {
+          #  $bootstrapOptions = $bootstrapOptions + " -EnableSanitizer address -EnableDebug"
+          # }
+          # if ($env:TILEDB_CI_TSAN -eq "ON") {
+          #  $bootstrapOptions = $bootstrapOptions + " -EnableSanitizer thread -EnableDebug"
+          # }
+          # static already added above as initial default
+          # if ($env:TILEDB_FORCE_BUILD_DEPS" -eq "ON") {
+          #   $bootstrapOptions = $bootstrapOptions + " -EnableBuildDeps"
+          # }
+          if ($env:TILEDB_WERROR -eq "OFF") {
+            $bootstrapOptions = $bootstrapOptions + " -DisableWerror"
+          }
+          
+          $bootstrapExpression = "& $env:BUILD_SOURCESDIRECTORY\bootstrap.ps1 " + $bootstrapOptions
+          Write-Host "bootstrapExpression: $bootstrapExpression"
+          Invoke-Expression $bootstrapExpression
+
+          if ($LastExitCode -ne 0) {
+            Write-Host "Bootstrap failed."
+            exit $LastExitCode
+          }
+
+          cmake --build $env:BUILD_BUILDDIRECTORY --config Release -j $env:NUMBER_OF_PROCESSORS

```

Comment:
Should probably redirect stderr here as below.

Reply:
done